test/common/stats/thread_local_store_test.cc

```diff
@@ -1381,7 +1381,7 @@ TEST_F(StatsThreadLocalStoreTestNoFixture, MemoryWithoutTlsRealSymbolTable) {
   TestUtil::forEachSampleStat(
       100, true, [this](absl::string_view name) { store_.counterFromString(std::string(name)); });
   EXPECT_MEMORY_EQ(memory_test.consumedBytes(), 688080); // July 2, 2020
-  EXPECT_MEMORY_LE(memory_test.consumedBytes(), 0.75 * million_);
+  EXPECT_MEMORY_LE(memory_test.consumedBytes(), 0.85 * million_);
```

Comment:
I do not know the implications of bumping these

Reply:
I believe this just indicates the predictable pattern isn't so predictable. But there's a second question of whether this particular sort of tests belongs in envoyproxy or should be part of the testing of the underlying cpplib/absl implementations.

Reply:
It's just as possible that the change occurred in our own TestUtil::MemoryTest logic?

Reply:
it pretty reliably fails with this change, so I feel like there must be something in tcmalloc affecting it, but I agree that it could be better as an upstream test if that's all we're really testing. I didn't check how on the cusp of exceeding the previous limits we were before